tests/ExampleApps/EntityAnimationDemo/main.cpp

```diff
@@ -0,0 +1,231 @@
+// Include Radium base application and its simple Gui
+#include <Gui/BaseApplication.hpp>
+#include <Gui/RadiumWindow/SimpleWindowFactory.hpp>
+
+// include the core geometry/appearance interface
+#include <Core/Geometry/MeshPrimitives.hpp>
+
+// include the Engine/entity/component/system/animation interface
+#include <Engine/FrameInfo.hpp>
+#include <Engine/Scene/EntityManager.hpp>
+#include <Engine/Scene/GeometryComponent.hpp>
+#include <Engine/Scene/System.hpp>
+
+// include the task animation interface
+#include <Core/Tasks/Task.hpp>
+
+// include the Camera and default camera manager interface
+#include <Engine/Scene/CameraComponent.hpp>
+#include <Engine/Scene/DefaultCameraManager.hpp>
+
+// include the viewer to activate the camera
+#include <Gui/Viewer/CameraManipulator.hpp>
+#include <Gui/Viewer/Viewer.hpp>
+
+// To terminate the demo after 4 seconds
+#include <QTimer>
+
+using namespace Ra;
+using namespace Ra::Core;
+using namespace Ra::Engine;
+
+/* ---------------------------------------------------------------------------------------------- */
+/*                      Custom window with some key events to switch from camera                  */
+/* ---------------------------------------------------------------------------------------------- */
+/**
+ * Simple custom windows for custom KeyEvent demonstration
+ */
+class DemoWindow : public Ra::Gui::SimpleWindow
+{
+    Q_OBJECT
+
+  public:
+    /// Reuse the SimpleWindow constructors
+    using Ra::Gui::SimpleWindow::SimpleWindow;
+
+    explicit DemoWindow( uint w = 800, uint h = 640, QWidget* parent = nullptr ) :
+        Ra::Gui::SimpleWindow::SimpleWindow( w, h, parent ) {}
+
+    //! [Configure Custom KeyEvent for the window]
+    void configure() override {
+        // <alt>+F will switch to default camera
+        // <alt>+C will toggle on cameras
+        // these customs events are only here for demonstration purposse.
+        // events expressed for an apple french keyboard
+        getViewer()->addKeyPressEventAction(
```

Comment:
These events look nice. Shall we add them to `SimpleWindows` as default ?

Reply:
The viewer, and so the `SimpleWindow`, already has an event to switch cameras in turn : `<ALT>+c` Key event. 
So, there is no need for these that are redundant with viewer switching.
If an application want a direct access to the camera, it is easy to add custom event, even on SimpleWindow, without defining a Custom Window as in this demo ... I'll simplify more the demo to do that.